documentation/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230506-implementation_plan_frontend_blurring_sensitive.md

```diff
@@ -0,0 +1,264 @@
+# 2023-05-06 Implementation Plan: Fetching, blurring sensitive results
+
+**Author**: @dhruvkb
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat for frontend experience
```

Comment:
It seems like Olga has opted out of this and asked me to review, which is fine, so just updating this part to reflect that :slightly_smiling_face: 

```suggestion
- [ ] @sarayourfriend
```